src/client/config/dev-cloud.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "useEndPoint": true,
```

Comment:
we can, but the url  changes based on the build number so we'd have to update the deployment to write out this file.